package.json

```diff
@@ -37,7 +37,8 @@
     "limiter": "2.1.0",
     "miscreant": "^0.3.2",
     "tron-format-address": "^0.1.8",
-    "typescript": "^4.7.4"
+    "typescript": "^4.7.4",
+    "web3": "^1.7.5"
```

Comment:
No need to install web3, u can use defillama/sdk instead

Reply:
Ah, nvm already mentioned

Reply:
ok i will try to do that change and will create new PR, aslo one more doubt do i need write all the info about our token again while submitting PR?, because its already listed and i am just adding one more contract.

Reply:
The changes are done and web3 is removed,please kindly check. @vrtnd